lms/djangoapps/discussion/static/discussion/js/spec/views/discussion_user_profile_view_spec.js

```diff
@@ -1,278 +1,49 @@
-define(
-    [
+/* globals Discussion, DiscussionCourseSettings */
+(function(define) {
+    'use strict';
+
+    define([
         'underscore',
         'jquery',
         'URI',
         'common/js/discussion/utils',
         'common/js/discussion/views/discussion_thread_profile_view',
+        'common/js/discussion/discussion',
         'common/js/spec_helpers/discussion_spec_helper',
         'discussion/js/views/discussion_user_profile_view'
     ],
-    function(_, $, URI, DiscussionUtil, DiscussionThreadProfileView, DiscussionSpecHelper, DiscussionUserProfileView) {
-        'use strict';
-
+    function(_, $, URI, DiscussionUtil, DiscussionThreadProfileView, Discussion,
+    DiscussionSpecHelper, DiscussionUserProfileView) {
         describe('DiscussionUserProfileView', function() {
-            var makeThreads, makeView;
-            beforeEach(function() {
-                DiscussionSpecHelper.setUpGlobals();
+            var createDiscussionUserProfileView;
+            createDiscussionUserProfileView = function() {
+                var discussionUserProfileView,
+                    discussion = DiscussionSpecHelper.createTestDiscussion({}),
+                    courseSettings = DiscussionSpecHelper.createTestCourseSettings();
+
+                setFixtures('<div class="discussion-user-profile-board"></div>');
                 DiscussionSpecHelper.setUnderscoreFixtures();
-                return spyOn(DiscussionThreadProfileView.prototype, 'render');
-            });
 
-            makeThreads = function(numThreads) {
-                return _.map(_.range(numThreads), function(i) {
-                    return {
-                        id: i.toString(),
-                        body: 'dummy body'
-                    };
+                discussionUserProfileView = new DiscussionUserProfileView({
```

Comment:
`return new DiscussionUserProfileView({`, then you won't need to create a new var for this on [20](https://github.com/edx/edx-platform/pull/14076/files#diff-e3711149464a12c68c0c4d9c3c2e545aR20)